src/test/java/com/github/arteam/dropwizard/http2/client/TestResource.java

```diff
@@ -11,11 +16,33 @@
  * @author Artem Prigoda
  */
 @Produces("application/json")
-@Path("greet")
+@Path("/")
 public class TestResource {
 
+    private ExecutorService executor = Executors.newCachedThreadPool();
+
     @GET
+    @Path("/greet")
     public Greeting greet() {
         return new Greeting("HTTP/2 server", "Hello, World!");
     }
+
+    @GET
+    @Path("/greet-chunk")
+    public ChunkedOutput<String> greetChunk() throws InterruptedException {
+        final ChunkedOutput<String> output = new ChunkedOutput<>(String.class);
+
+        executor.execute(() -> {
+            try (ChunkedOutput<String> out = output) {
+                for (int i = 0; i < 5; i++) {
+                    Thread.sleep(500L);
+                    out.write("Hello world, I'm a HTTP/2 chunk server");
+                }
+            } catch (IOException | InterruptedException ignored) {
```

Comment:
I think it would be better to separate `IOException` and `InterruptedException` ans call `Thread.currentThread().interrupt()`